src/index.js

```diff
@@ -142,6 +144,34 @@ function validateForm() {
         let totalPriceForTheTrip = totalPricePerPerson * desiredTrip.gatheredTravelers;
         domUpdates.displayTripCost((totalPriceForTheTrip * 1.1))
     }
+
+    // findNewTripID(allTrips) {
+    //     let highestID = allTrips.sort((lowerTripId, higherTripId) => {
+    //         return lowerTripId.id + higherTripId.id
+    //     })[0].id;
+    //     return (highestID + 1)
+    // }
+
+    function buildATrip(desiredTrip) {
+        // let promisedAllTrips = fetcher.fetchTripsForAUser();
+
+        requestedTripObject = {
+            id: Date.now(),
+            userID: user.id,
+            destinationID: desiredTrip.gatheredDestination,
+            travelers: desiredTrip.gatheredTravelers,
+            date: desiredTrip.gatheredDate,
+            duration: desiredTrip.gatheredDuration,
+            status: 'pending',
+            suggestedActivities: []
```

Comment:
If I ended up getting to it4 (I wont), but would I want to be able to pass in some kind of "desired activities"? Is this where this would go?